packages/calypso-e2e/src/lib/flows/start-site-flow.ts

```diff
@@ -166,22 +166,8 @@ export class StartSiteFlow {
 	 *
 	 * @param {string} themeName Name of theme, e.g. "Zoologist".
 	 */
-	async previewTheme( themeName: string ): Promise< void > {
-		if ( envVariables.VIEWPORT_NAME === 'desktop' ) {
-			await this.page.hover( selectors.individualThemeContainer( themeName ) );
-			await this.page.click( selectors.previewThemeButtonDesktop( themeName ) );
-		} else {
-			await this.page.click( selectors.previewThemeButtonMobile( themeName ) );
-		}
-	}
-
-	/**
-	 * Get a Frame handle for the iframe holding the theme preview.
-	 *
-	 * @returns The Frame handle for the theme preview iframe.
-	 */
-	async getThemePreviewIframe(): Promise< Frame > {
-		const elementHandle = await this.page.waitForSelector( selectors.themePreviewIframe );
-		return ( await elementHandle.contentFrame() ) as Frame;
+	async selectTheme( themeName: string ): Promise< void > {
+		const locator = this.page.locator( selectors.individualThemeContainer( themeName ) );
+		await locator.click();
```

Comment:
`previewTheme` is not used anywhere, and the notion of hover-to-preview-theme does not exist anymore in the current version of the design picker.

![image](https://user-images.githubusercontent.com/6549265/181641012-0cb7d4d5-6681-4a67-a874-35e6d6271855.png)